connector/server/survivalanalysis/timepoints/sequential_data.go

```diff
@@ -0,0 +1,107 @@
+package timepoints
```

Comment:
This seems to be missing unit tests

Reply:
all functions are tested once in buildtimepoint but I will add one test for each